manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -158,6 +158,9 @@ spec:
             memcachedSlabPageSize:
               description: 'Memcached max item size (default: 1m, min: 1k, max: 1024m)'
               type: string
+            miqServerGuid:
```

Comment:
let's leave out the miq for prodcutization reasons (I know it's related to MiqServer model, but end users don't see that; they only see "Server" in the UI).